### ISSUE TITLE
The x-axis labels go out of the boundary after re-rendering

### ISSUE THREAD

**Issue by arnajdas92** [carbon-design-system/carbon-charts#121]
#### I am submitting a...

- [x] Design defect

#### charts version:
0.7.11

### Issue description
The x-axis labels go out of the boundary after re-rendering. As visible in the screenshots, the x-axis labels at first wrap around properly. But after the graph re-renders they go out of border.

### Screenshot or recording
<img width="384" alt="screenshot 2019-01-02 at 2 23 35 pm" src="https://user-images.githubusercontent.com/7128486/50585284-3c3e9e80-0e9a-11e9-97c1-872bcbe09134.png">
<img width="386" alt="screenshot 2019-01-02 at 2 23 45 pm" src="https://user-images.githubusercontent.com/7128486/50585285-3c3e9e80-0e9a-11e9-965a-95fc19aee9c7.png">

**Comment by t-mullen** [comment]
This breaks since we rerender the chart in a transition - which removes all the formatting on tick values we do.

@theiliad Can you explain why we do `.call(xAxis)` here? Removing it fixes this issue. https://github.com/carbon-design-system/carbon-charts/blob/master/packages/core/src/base-axis-chart.ts#L219

**Comment by theiliad** [comment]
@t-mullen Thanks for pointing out the issue. I'll get back to you

**Comment by theiliad** [comment]
As far as I remember, you would have to use `.call` to make sure new axis elements will be added

**Comment by kalai001** [comment]
@theiliad , Another related observation. The text in the graph doesn't wrap in the initial rendering also, if the graph is loaded with default width and height. Ideally, the text wrapping be should be determined on the availability of break characters (space).

**Comment by pjena98** [comment]
We are waiting for a fix and request ETA

**Comment by theiliad** [comment]
@pjena98 It's tough to figure out a solution that wouldn't cover the chart.

We've handed off the issue to the design team to come up with a solution. You'll hear back soon

**Comment by theiliad** [comment]
This was addressed in #275
